### ISSUE TITLE
Mesh errors

### ISSUE THREAD

**Issue by ghost** [CruzR/Blender-MDL#16]
I've imported a character 3d mesh and it turned out like this:
![untitled](https://cloud.githubusercontent.com/assets/5841766/4265175/bd23d766-3c41-11e4-975a-cdd0f3585e3c.png)

Any suggestions?

**Comment by CruzR** [comment]
Yeah, I haven't really worked on this plugin for a long time, and it is currently kind of broken (see also ticket #15. I'll see what I can do.

**Comment by DerekX** [comment]
Just to give you a reminder, it'll really help if you could make it so that we could also import the models' animation sequences. There are a lot of awesome WC3 models on Hive Workshop.
